src/engine/Docs/Animations.md

```diff
@@ -30,4 +30,5 @@ game.start(loader).then(function () {
 
 You can add [["Drawing/SpriteEffects"|sprite effects]] to an animation through methods
 like [[Animation.invert]] or [[Animation.lighten]]. Keep in mind, since this
-manipulates the raw pixel values of a [[Sprite]], it can have a performance impact.
\ No newline at end of file
+manipulates the raw pixel values of a [[Sprite]], it can have a performance impact.
+[[Animetion.loop]] is now set to true by default.
```

Comment:
Typo and rephrase as "Animations will loop by default. You can use [[Animation.loop]] to change this behavior."